spec/wasSeedPreassembly/fixtures/workspace/aa/111/aa/1111/aa111aa1111/metadata/descMetadata.xml

```diff
@@ -1,52 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
this is a generated file and should never have been in source control in the first place